application/common/components/phoneVerification/Base.php

```diff
@@ -112,8 +112,12 @@ public function format($phoneNumber)
                 return sprintf('%s %s', $insights['country_prefix'], $insights['national_format_number']);
             } else {
                 throw new \Exception(
-                    'Unable to verify phone number for formatting, please check the number and try again. ' .
-                    'Error code: ' . $insights['status'],
+                    \Yii::t(
+                        'app',
+                        'Unable to verify phone number for formatting, please check the number and try again. ' .
+                            'Error code: {code}',
```

Comment:
This is a minor, somewhat subjective comment, but we typically don't do a full indent on a string-continuation. So rather than this...
```php
/Yii::t(
    'app',
    'A rather long string that ' .
        'continues on a 2nd line.'
)
```
... we might do this:
```php
/Yii::t(
    'app',
    'A rather long string that '
    . 'continues on a 2nd line.'
)
```
That seems to make it easier to tell (without having to go hunting for the end of the line) that it's a continuation of the string on the previous line and not some further function call causing an additional indentation level.

Reply:
Thanks. I'll make that adjustment.